leetcode/tree.md

```diff
@@ -0,0 +1,24 @@
+# Tree
```

Comment:
Your pull request: LEETCODE-1022 Added binary-tree-inorder-traversal


Your commit: LEETCODE-1022 Added binary-tree-inorder-traversal

Your commit: LEETCODE-1022 Refactored binary-tree-inorder-traversal
Action of your comment must be from ['Added', 'Deleted']